modules/isd/include/ResidueProteinProximityRestraint.h

```diff
@@ -0,0 +1,83 @@
+/**
+ *  \file IMP/isd/ResidueProteinProximityRestraint.h
+ *  \brief Restraint a selection of particles (eg. a residue or 
+ *  segment) to be within  a certain distance of a second 
+ *  selection of particles (eg. a protein). 
+ *  Use to model data from mutagenesis experiments that disrupt
+ *  protein-protein interactions.
+ *
+ *  Copyright 2007-2022 IMP Inventors. All rights reserved.
+ *
+ */
+
+#ifndef IMPISD_RESIDUE_BINDING_RESTRAINT_H
+#define IMPISD_RESIDUE_BINDING_RESTRAINT_H
+#include "isd_config.h"
+#include <IMP/container/ListSingletonContainer.h>
+#include <IMP/container_macros.h>
+#include <IMP/PairContainer.h>
+#include <IMP/Restraint.h>
+#include <boost/unordered_map.hpp>
+
+IMPISD_BEGIN_NAMESPACE
+
+//! Apply harmonic restraint between a residue or fragment and a
+//! protein.
+
+class IMPISDEXPORT ResidueProteinProximityRestraint : public Restraint {
+public:
+  //! Create the restraint.
+ ResidueProteinProximityRestraint(IMP::Model *m,
```

Comment:
`IMP::` prefix should be unnecessary here.

Reply:
Done